docs/spark-queries.md

```diff
@@ -394,3 +394,22 @@ spark.read.format("iceberg").load("db.table.files").show(truncate = false)
 // Hadoop path table
 spark.read.format("iceberg").load("hdfs://nn:8020/path/to/table#files").show(truncate = false)
 ```
+
+### Time Travel with Metadata Tables
+
+To inspect a tables's metadata with the time travel feature:
+
+```sql
+-- get the table's file manifests at timestamp Sep 20, 2021 08:00:00
+SELECT * FROM prod.db.table.manifests TIMESTAMP AS OF '2021-09-20 08:00:00';
```

Comment:
Nit: can we remove the end semicolon for consistency?

Reply:
I see some places end with this `;` and some other places do not in this doc. I do not have specific preference for example code. But it seems a bit clearer for a code block with multiple SQL statement (like here). Do you think it's a good idea to make all places end with `;` instead?

Reply:
You are right, I didn't notice it.

Reply:
I make all other SQL example end with `;` to make this style consistent and embarrassingly clear that the statement is complete.